src/content/developers/docs/oracles/index.md

```diff
@@ -8,6 +8,211 @@ incomplete: true
 
 Oracles are a data feed that connect Ethereum to off-chain, real-world information, so you can query data in your smart contracts. For example, prediction market dapps use oracles to settle payments based on events. A prediction market may ask you to bet your ETH on the next president of the United States. They'll use an oracle to confirm the outcome and pay out to the winners.
 
+## Prerequisites {#prerequisites}
+
+Make sure you're familiar with [nodes](/developers/docs/nodes-and-clients/), [consensus mechanisms](/developers/docs/consensus-mechanisms/), and [smart contract anatomy](/developer/docs/smart-contracts/anatomy/), specifically events.
+
+## What is an oracle {#what-is-an-oracle}
+
+Oracles act as a bridge between the blockchain and the real world. They allow you to query APIs to get information into your smart contracts. This could be anything from price information to weather reports.
```

Comment:
```suggestion
An oracle is a bridge between the blockchain and the real world. They act as on-chain APIs you can query to get information into your smart contracts. This could be anything from price information to weather reports.
```